eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -159,10 +159,34 @@ jobs:
 
       - template: ../steps/upload-repository-on-failure.yml
 
+      - task: Maven@3
```

Comment:
nit: Move this task to run immediately after the `'Run tests: $(DisplayName)'` task is complete.

Reply:
Done, thanks Srikanta